tryhackme/user.py

```diff
@@ -58,7 +58,4 @@ def _from_data(self, data):
     
     def _sync(self):
         self.message_groups = [MessageGroup(state=self._state, data=group) for group in self._state.http.get_all_group_messages()]
```

Comment:
# Issue:
- unnessesery onload overhead (since is wont be used much)
- dynamic data, meaning that the user can join and leave groups at runtime
# ToDo:
- set `self.message_groups` to class only `self._message_groups`
- move group building to State machine #7